src/Components/Navbar.jsx

```diff
@@ -85,10 +111,30 @@ const Navbar = () => {
           variant={"unstyled"}
           placeholder="Search for products"
           htmlSize={50}
+          onChange={(e) => searchItems(e.target.value)}
```

Comment:
Never call onChange or onClick handlers here. wrap it inside the useCallback or useMemo .